other/maximum_subarray.py

```diff
@@ -1,26 +1,30 @@
-def max_subarray(nums: list[int]) -> int:
-    """
-    Returns the subarray with maximum sum
-    >>> max_subarray([1,2,3,4,-2])
+from collections.abc import Sequence
```

Comment:
```suggestion
from typing import Sequence
```

Before Python 3.8 (or 3.9) this would be an exception.

Reply:
Committing this suggestion is creating an issue whilst running the test associated with ```pyupgrade```.

Reply:
Looks like after they got the support of [] in Python 3.9, typing became the less favorable place to import them.

Reply:
Yes. Now, the pre-commit hooks following the failed ```pyupgrade``` test are designed in a way that it makes the suggestion null and void.